eppo/src/androidTest/java/cloud/eppo/android/EppoClientTest.java

```diff
@@ -62,44 +62,50 @@ public class EppoClientTest {
       "https://us-central1-eppo-qa.cloudfunctions.net/serveGitHubRacTestFile";
   private static final String INVALID_HOST = "https://thisisabaddomainforthistest.com";
   private final ObjectMapper mapper = new ObjectMapper().registerModule(module());
-
   private AssignmentLogger mockAssignmentLogger;
 
   private void initClient(
       String host,
       boolean throwOnCallbackError,
       boolean shouldDeleteCacheFiles,
       boolean isGracefulMode,
+      boolean obfuscateConfig,
+      @Nullable EppoHttpClient httpClientOverride,
+      @Nullable ConfigurationStore configurationStoreOverride,
       String apiKey) {
     if (shouldDeleteCacheFiles) {
       clearCacheFile(apiKey);
     }
     mockAssignmentLogger = mock(AssignmentLogger.class);
     CountDownLatch lock = new CountDownLatch(1);
 
+    InitializationCallback callback =
+        new InitializationCallback() {
+          @Override
+          public void onCompleted() {
+            Log.w(TAG, "Test client onCompleted callback");
```

Comment:
I wonder if while moving this we should change that log to info (`Log.i()`) from warn.

Reply:
good call, I can do it